### ISSUE TITLE
Fix the horrendous thing which is the carousel

### ISSUE THREAD

**Issue by Robinerra** [Robinerra/Portfolio-Website#14]
The skills/hobbies/techstack slide is terrible from a code beauty perspective, how did I even allow this to exist?

**Comment by Robinerra** [comment]
Additionally it's paragraphs in a div, not accessible to screen readers because we want them to know it's a list of things

**Comment by Robinerra** [comment]
Patched in PR #15
